bigtable/use-cases/fraudDetection/src/main/java/bigtable/fraud/beam/LoadDataset.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package bigtable.fraud.beam;
+
+import bigtable.fraud.utils.CustomerDemographics;
+import bigtable.fraud.utils.TransactionDetails;
+import bigtable.fraud.utils.RowDetails;
+import com.google.cloud.bigtable.beam.CloudBigtableIO;
+import com.google.cloud.bigtable.beam.CloudBigtableTableConfiguration;
+import com.google.common.base.Preconditions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.io.TextIO;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.MapElements;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.TypeDescriptor;
+import org.apache.hadoop.hbase.client.Mutation;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// Load customer demographics and history into Cloud Bigtable.
+public final class LoadDataset {
+
+  /**
+   * Hiding the constructor.
+   */
+  private LoadDataset() {
+  }
+
+  /**
+   * a Logger object to help logging details.
+   */
+  private static final Logger LOGGER = LoggerFactory.getLogger(
+      LoadDataset.class);
+
+  /**
+   * Convert a RowDetails into a Mutation.
+   * The row key is the first member variable in the class that
+   * inherits RowDetails.
+   */
+  private static final DoFn<RowDetails, Mutation> MUTATION_TRANSFORM =
```

Comment:
This makes the below stuff a lot cleaner, thank you! It might be nice to still keep these in a separate file in the utils directory as you had before and then just import them